Tests/TestKernel.php

```diff
@@ -141,7 +141,18 @@ protected function configureContainer(ContainerBuilder $container, LoaderInterfa
                 ],
                 'platform_service' => SqlitePlatform::class,
             ],
-            'orm' => null,
+            'orm' => [
+                'auto_mapping' => true,
+                'mappings' => [
+                    'App' => [
+                        'type' => 'annotation',
+                        'dir' => '../docs/resources/entities',
```

Comment:
@HypeMC or @X-Coder264; In order to fix the tests, perhaps one of you could figure out what the correct doctrine mapping source is in the project to the demo entities I created in `docs/resources/entities`? It seems useless to re-create these files just for testing.

I would like to have your opinion on this matter.